ajax/libs/Sharrre/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Sharrre",
+  "filename": "jquery.sharrre.min.js",
+  "version": "2.0.1",
+  "description": "Make your sharing widget! Sharrre is a jQuery plugin that allows you to create nice widgets sharing for Facebook, Twitter, Google Plus (with PHP script) and more.",
+  "author": "Julien Hany",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Julienh/Sharrre.git"
+  },
+  "homepage": "http://sharrre.com",
+  "keywords": [
+    "jQuery",
+    "plugin",
+    "share",
+    "widget"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Julienh/Sharrre.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(test.css)"
```

Comment:
I have a question about that should we host `style.css`?
Because I can't find any doucument said that people should use add it.
And I saw the css code of [`demo1`](https://github.com/Julienh/Sharrre/blob/master/dist/style.css#L1), demo2.
I think these just for the demo as the homepage.

What do you think?
Or maybe we should discuss with the author?